microk8s-resources/actions/common/utils.sh

```diff
@@ -1077,6 +1061,87 @@ server_cert_check() {
   openssl x509 -in "$SNAP_DATA"/certs/server.crt -outform der | sha256sum | cut -d' ' -f1 | cut -c1-12
 }
 
+########################################################################
```

Comment:
I know that the formatting is inconsistent in this file but could we please follow the function documentation scheme of for example `create_kubeconfig_x509`, where the description and arguments go under the function name?